### ISSUE TITLE
Invalid default values of gitImage and busyboxImage in the release Helm Chart

### ISSUE THREAD

**Issue by gw-lling** [kubevela/terraform-controller#369]
We noticed that the release helm chart has incorrect version for gitImage and busyboxImage.
Both are set to same version of chart version, which is not valid.
We believe the issue is at [github action](https://github.com/kubevela/terraform-controller/blob/master/.github/workflows/chart.yml#L28) for release where is replacing all `latest` to the chart version.

We chart version we are using is 0.7.11.

```
replicaCount: 1

image:
  repository: oamdev/terraform-controller
  tag: v0.7.11
  pullPolicy: Always

gitImage: alpine/git:v0.7.11
busyboxImage: busybox:v0.7.11
terraformImage: oamdev/docker-terraform:1.1.5
controllerNamespace: ""

# "{\"nat\": \"true\"}"
jobNodeSelector: ""
jobBackoffLimit: ""

resources:
  limits:
    cpu: "500m"
    memory: "500Mi"
  requests:
    cpu: "250m"
    memory: "250Mi"

backend:
  namespace: vela-system

githubBlocked: "'false'"

featureGates:
  # Enable the feature of allowing to delete a configuration whose cloud resources is not fully provisioned, or error happens
  # This guarantees that the partial cloud resources will be deleted when the configuration is deleted
  # Default value is true
  AllowDeleteProvisioningResource: true
```

**Comment by anoop2811** [comment]
Looks like https://github.com/kubevela/terraform-controller/blob/master/.github/workflows/chart.yml#L28 is the culprit which updates the latest tag for alpine and busybox as well. cc: @wonderflow  @Somefive

**Comment by anoop2811** [comment]
cc: @wonderflow @Somefive

**Comment by chivalryq** [comment]
Fixed in #730
